lib/init.coffee

```diff
@@ -1,8 +1,54 @@
+{CompositeDisposable} = require 'atom'
+
 module.exports =
   config:
-    xmllintExecutablePath:
+    executablePath:
       type: 'string'
-      default: ''
-
+      title: 'xmllint Executable Path'
+      default: 'xmllint'
   activate: ->
-    console.log 'activate linter-xmllint'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-xmllint.xmllintExecutablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+    @executablePath = '/usr/bin/xmllint'
+  deactivate: ->
+    @subscriptions.dispose()
+
+  lintFile: (filePath)->
+    helpers = require 'atom-linter'
+    params = ['--noout', filePath]
+    return helpers.exec(@executablePath, params, {stream: 'stderr'})
```

Comment:
Afaik `xmllint` has no config file nor does it use the cwd anywhere. Also this package didn't set a cwd before so I would rather not set one if there is no reason to do so.